doc(Tabs): update FAQ (#5775)
ant-design/ant-design-mobile: src/components/tabs/index.en.md

### How does Tabs work with Swiper, PullToRefresh, and InfiniteScroll to implement a complex information flow interface?

You can refer to this [demo](https://codesandbox.io/s/mystifying-glitter-knpc7u?file=/src/components/getPullToRefreshlData.tsx).

### Question about the type of Tabs.Tab key?

> Because `React` internally converts the incoming `key` into `string`, the type of `key` here should actually conform to `React.Key` can be used.

Reviewer: ```suggestion
`React` will convert the incoming `key` into `string`, the type of `key` only needs to match `React.Key`. 
```
